### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by harmony-weblate**
Translations update from [Weblate](https://translate.harmony.tf) for [Northstar/Northstar Client Localisation](https://translate.harmony.tf/projects/northstar/client/).

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
@ASpoonPlaysGames what happened to compile check? ^^"

**Comment by ASpoonPlaysGames** [comment]
I have no idea, I haven't touched it

**Comment by ASpoonPlaysGames** [comment]
> @ASpoonPlaysGames what happened to compile check? ^^"

Seems to be a github issue? appears to be working fine now...

File: Northstar.Client/mod/resource/northstar_client_localisation_french.txt [hunk 1]
```diff
@@ -341,5 +341,17 @@ Choisissez Oui si vous êtes d'accord. Ce choix peut être modifié à tout inst
...
+		"PROGRESSION_TOGGLE_DISABLED_BODY"        "Les Titans, Armes, Factions, Skins et autres seront débloqués par la monté en niveau ou par leur achats en mérites.\n\nPeut être changé à n'importe que moment dans le salon multijoueurs.\n\n^CC000000Warning : Si vous équiper des objets que vous n'avez pas encore débloqués, ils seront déséquipés !"
+		"PROGRESSION_ENABLED_BODY"        "^CCCC0000La progression a été activée.^\n\nLes Titans, Armes, Factions, Skins et autres seront débloqués par la monté en niveau ou par leur achats en mérites.\n\nPeut être changé à n'importe que moment dans le salon multijoueurs."
+		"TOGGLE_PROGRESSION"        "Activer la progression"
+		"Y_BUTTON_TOGGLE_PROGRESSION"        "%[Y_BUTTON]% Activer la progression"
+		"PROGRESSION_ANNOUNCEMENT_BODY"        "^CCCC0000Le système de progression peut être activé !^\n\nNorthstar supporte désormais le système de progression du jeu original, vous permettant de choisir si vous souhaitez débloquer les armes, skins, titans etc. en gagnant des niveaux et en complétant des défis.\n\nVous pouvez activer la progression en utilisant le bouton en bas de l'écran d'accueil.\n\nCeci peut être changé à tout moment."
```
Comment (GeckoEidechse) [inline Northstar.Client/mod/resource/northstar_client_localisation_french.txt:355]: @Alystrasz the stuff produced by weblate looks a bit odd here. Maybe just a custom encoding I'm unaware off? You wanna test the PR by any chance? ^^"

File: Northstar.Client/mod/resource/northstar_client_localisation_spanish.txt [hunk 2]
```diff
@@ -85,11 +85,11 @@ Presiona Sí al estar de acuerdo. Esta opcion se puede cambiar en el menú de mo
 		"aitdm_archer_grunts" "Soldado Archer"
 
 		// northstar.custom localisation is just deciding not to work, so putting it here for now
-		"PL_sbox" "Sandbox"
-		"PL_sbox_lobby" "Vestíbulo de Sandbox"
-		"PL_sbox_desc" "como gmod pero peor"
+		"PL_sbox" "Custom"
```
Comment (GeckoEidechse) [inline Northstar.Client/mod/resource/northstar_client_localisation_spanish.txt:88]: This change feels a bit odd tbh. Maybe `Sandbox` gamemodes are commonly called `Custom` in Spanish for some reason?

File: Northstar.Client/mod/resource/northstar_client_localisation_french.txt (hunk 1 above)
Comment (ASpoonPlaysGames) [reply Northstar.Client/mod/resource/northstar_client_localisation_french.txt:355]: What exactly looks weird about it?

Comment (Alystrasz) [reply Northstar.Client/mod/resource/northstar_client_localisation_french.txt:355]: Other than typos (that I just addressed through weblate), I don't see what's wrong here

Comment (GeckoEidechse) [reply Northstar.Client/mod/resource/northstar_client_localisation_french.txt:355]: I mean the `^CCCC0000` part or `PROGRESSION_ANNOUNCEMENT_BODY` for example. Or is that also in the English version? ^^

Comment (ASpoonPlaysGames) [reply Northstar.Client/mod/resource/northstar_client_localisation_french.txt:355]: Indeed it is, localisation file syntax for coloured text. There is a matching ^ at the end of the coloured text

File: Northstar.Client/mod/resource/northstar_client_localisation_spanish.txt (hunk 2 above)
Comment (GeckoEidechse) [reply Northstar.Client/mod/resource/northstar_client_localisation_spanish.txt:88]: Eh, I'm just gonna resolve this and assume it's fine. If it's not someone will probably step in and fix it again anyway ^^

**Review by GeckoEidechse — APPROVED** [review]
Concerns raised in review about translations have been more or less addressed. I'm still not sure about the `Sandbox` -> `Custom` change but I'm not gonna block the PR over that.

If it's wrong, someone will probably step in to fix it again anyway ^^

**Not tested as it's only translation changes.**

**Pull request merged**